plugins/theme-modern/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@blockly/theme-modern",
+  "version": "0.20200417.0",
```

Comment:
Are you planning on bumping to 1 before publishing?

Reply:
Yeah I wonder if we should have all the plugins start at 1.

Reply:
1.date.0? Or just 1.0.0?

Reply:
I think we standardized on major.date.patch to match google-closure / Blockly.